src/librustc_mir/const_eval.rs

```diff
@@ -567,9 +573,7 @@ fn validate_and_turn_into_const<'tcx>(
         let err = error_to_const_error(&ecx, error);
         match err.struct_error(ecx.tcx, "it is undefined behavior to use this value") {
             Ok(mut diag) => {
-                diag.note("The rules on what exactly is undefined behavior aren't clear, \
-                    so this check might be overzealous. Please open an issue on the rust compiler \
-                    repository if you believe it should not be considered undefined behavior",
+                diag.note(undefined_behavior_error()
                 );
```

Comment:
Can this close bracket be moved up to the last line?

Reply:
OK